test/e2e/specs/test.spec.js

```diff
@@ -16,46 +16,53 @@ describe('Vaults UI Test Cases', () => {
           walletName: 'user1',
         });
       } else {
-        const walletAddress = {
-          value: null,
-        };
         cy.setupWallet({
           createNewWallet: true,
           walletName: 'my created wallet',
+          selectedChains: ['Agoric'],
         });
```

Comment:
I am assuming this is done to get a wallet address?

Reply:
yes